test/src/utils/artifact/generateAragonArtifact.test.ts

```diff
@@ -16,6 +16,7 @@ describe('ast > generateAragonArtifact', () => {
     if (arapp && flatCode && artifact) {
       it(`Should generate artifact.json - ${appName}`, () => {
         const newArtifact = generateAragonArtifact(
+          'mainnet',
```

Comment:
Just wanted to sanity check this, since it's changing something unrelated to just re-ordering imports and is potentially dangerous given we're changing the entire argument order.

Reply:
Function signature re-written in #120 to use name args and prevent this errors